docs/reference/solidreference.md

```diff
@@ -16,3 +16,16 @@ The only constraint when referring to a [Solid](solid.md) is that both [Solid](s
 
 - `solidName`: This field specifies either the static environment (if the value is `<static environment>`) or the name of an existing [Solid](solid.md) node to be linked with the [Joint](joint.md)'s closest upper [Solid](solid.md) node.
 Referring to the [Joint](joint.md) closest upper [Solid](solid.md) node or to a [Solid](solid.md) node which has no common upper [Solid](solid.md) with the [Joint](joint.md) is prohibited.
+
+### Mechanical Loop Example
+
+```
+ DEF FRONT_AXLE_HINGE HingeJoint {
+   jointParameters HingeJointParameters {
+     anchor -0.08 -0.038 -0.11
+   }
+   endPoint SolidReference {
+     solidName "front axle"
+   }
+ }
```

Comment:
```suggestion
HingeJoint {
  jointParameters HingeJointParameters {
    anchor 0 0 1
  }
  endPoint Solid {
    name "front axle"
  }
}
HingeJoint {
  jointParameters HingeJointParameters {
    anchor 0 0 -1
  }
  endPoint SolidReference {
    solidName "front axle"
  }
}
```

For a more complete example, it is better to show the 'referred' solid in the example.

Reply:
Okay, thank you, I made changes.